js/functional/palindrome.js

```diff
@@ -17,7 +26,22 @@ function isPalindromeUsingReverse(value) {
  */
 function isPalindromeUsingLoop(value) {
   // write your code here
-  return true;
+  value = parseString(value);
+
+  const cleanedValue = value.toLowerCase().replace(/[^a-z0-9]/g, "");
+
+  // Initialize length of cleanedValue
+  const length = cleanedValue.length;
+
+  // Loop through the characters to check for palindrome
+  for (let i = 0; i < length / 2; i++) {
```

Comment:
I prefer to use "index" than just "i" as the name for this loop variable because it's much more distinctive to understand for the other.